packages/api-client-core/src/GadgetRecordList.ts

```diff
@@ -2,8 +2,8 @@
 /* eslint-disable @typescript-eslint/require-await */
 import { GadgetRecord } from "./GadgetRecord";
 import { InternalModelManager } from "./InternalModelManager";
-import { ModelManager } from "./ModelManager";
-import type { PaginationOptions } from "./operationRunners";
```

Comment:
👍 will change.

I think auto import always assumes you want to use something as a value. Weird, since I don't think `PaginationOptions` ever could, but I guess there's always the possibility of an import having side effects 😬